.github/workflows/daily-deploy-production.yml

```
@@ -96,10 +97,58 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
+  build:
+    name: Build
+    runs-on: self-hosted
+    needs: [set-env, notify-start, validate-build-status]
+    
+    env:
+      NODE_EXTRA_CA_CERTS: /etc/ssl/certs/VA-Internal-S2-RCA1-v1.cer.pem
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

We need to check out the correct sha for the build
```suggestion
        uses: actions/checkout@v2
        with:
          ref: ${{ needs.set-env.outputs.COMMIT_SHA }}
```